service/src/main/java/uk/gov/hmcts/reform/fpl/service/orders/prepopulator/modifier/ManageOrdersCaseDataFixer.java

```diff
@@ -48,4 +50,16 @@ public CaseData fix(CaseData caseData) {
 
         return caseData;
     }
+
+    public CaseDetails fixAndRetriveCaseDetails(CaseDetails caseDetails) {
+        if (!isNull((caseDetails.getData().get("manageOrdersOperation")))) {
+            String operation = caseDetails.getData().get("manageOrdersOperation").toString();
+
+            if (!AMEND.toString().equals(operation)) {
```

Comment:
I would recommend using the name method here, toString can be overwritten here so it has the possibility of not being `AMEND`

Reply:
makes sense :D